Orange/widgets/data/tests/test_owrank.py

```diff
@@ -556,7 +556,7 @@ def test_concurrent_cancel(self):
 class TestRankModel(GuiTest):
```

Comment:
Could you, please, also move this test to `Orange.widgets.tests.test_gui`?

Reply:
Done. I just wasn't sure whether to rename the test...

Reply:
I apologize for being such a nazi. We, that is pylint and I, think that you should also clean and rearrange your imports.

Reply:
I know. I was waiting for hints from the lint run. I don't know how to run it locally so.... 🤷‍♀️